content/applications/sales/crm/optimize/partner_autocomplete.rst

```diff
@@ -1,48 +1,70 @@
-===================================================
-Enrich your contacts base with Partner Autocomplete
-===================================================
+==============================================
+Enrich contacts base with partner autocomplete
+==============================================
 
-Parter Autocomplete helps you to enrich your contacts database with corporate data. Select one of
-the companies suggested in the dropdown, and quickly get all the information you need.
+*Partner autocomplete* helps to enrich the contacts database with corporate data. Select one of the
+companies suggested in the drop-down menu, and quickly get all the information.
+
+The information provided by partner autocomplete can include general information about the business
+(including full business name and logo), social media accounts, :guilabel:`Company type`,
+:guilabel:`Founded` information, :guilabel:`Sectors` information, the number of
+:guilabel:`Employees`, :guilabel:`Estimated revenue`, :guilabel:`Phone` number,
+:guilabel:`Timezone`, and :guilabel:`Technologies Used`.
+
+.. important::
+   When getting a company's contact information make sure to be aware of the latest EU regulation,
+   get more information about General Data Protection Regulation on `Odoo GDPR
+   <http://odoo.com/gdpr>`__. In Odoo, individual contact information cannot be searched for, using
+   the partner autocomplete feature.
 
 Configuration
 =============
 
 Go to :menuselection:`Settings --> Contacts` and activate the *Partner Autocomplete* feature.
 
-.. image:: partner_autocomplete/settings_partner_autocomplete.png
+.. image:: partner_autocomplete/settings-partner-autocomplete.png
    :align: center
    :alt: View of settings page and the activations of the feature in Odoo
 
-Enrich your contacts with corporate data
-========================================
+Enrich contacts with corporate data
+===================================
 
-| From any module, once you start to create a new contact typing the name of it, the system
-  will suggest a potential match. If you select it, the contact will be populated with corporate
-  data.
-| For example, after typing *Odoo*, you will get the following information:
+From any module, once a new company contact is entered and the user is typing the name of it, the
+system will suggest a potential match. If it is selected, the contact will be populated with
+corporate data.
 
-.. image:: partner_autocomplete/odoo_autocomplete.png
+For example, after typing *Odoo*, the following information will populate:
+
+.. image:: partner_autocomplete/odoo-autocomplete.png
    :align: center
    :alt: Creating a new contact in Odoo
 
-.. image:: partner_autocomplete/odoo_info_autocomplete.png
+.. image:: partner_autocomplete/odoo-info-autocomplete.png
```

Comment:
You may want to add a line (or even a caption) indicating that this information is included in the chatter thread for the contact. (Sorry I didn't catch this on the first round)